mmv1/third_party/terraform/services/bigquery/resource_bigquery_table.go

```diff
@@ -774,6 +776,7 @@ func ResourceBigQueryTable() *schema.Resource {
 				Description:      `A JSON schema for the table.`,
```

Comment:
I think this is the schema field that should be mutually exclusive?

Reply:
Thanks for the catch, fixed